src/util.ts

```diff
@@ -1032,7 +1034,20 @@ export class Util {
                     case 'void':
                         return new VoidType();
                 }
+                if (allowCustomType) {
+                    return new CustomType(token.text);
+                }
+        }
+    }
+
+    /**
+    * Convert a BscType to a type string
+    */
+    public bscTypeToTypeString(bscType: BscType) {
```

Comment:
I feel like this should exist on the type itself. Like `BscType.toTypeString()`. Most of the bsc types could just return `toString()`.